docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,82 @@
+| `RightTarget`          | View            | Gets or sets the right side edge of the child view relative to the given target view.|
+| `TopTarget`            | View            | Gets or sets the top side edge of the child view relative to the given target view.|
+| `BottomTarget`         | View            | Gets or sets the bottom side edge of the child view relative to the given target view.|
+| `LeftRelativeOffset`   | float           | Gets or sets the relative offset for the left target. <br /> When the value is `0`, the left edges of the left target and child view are aligned.<br/> When the value is `1`, the left edge of the child view is aligned to the right edge of the left target. |
+| `RightRelativeOffset`  | float           | Gets or sets the relative offset for the right target. <br /> When the value is `0`, the right edge of the child view is aligned to the left edge of the right target.<br/> When the value is `1`, the right edges of the right target and child view are aligned.
+| `TopRelativeOffset`    | float           | Gets or sets the relative offset for the top target. <br /> When the value is `0`, the top edges of the top target and child view are aligned.<br/> When the value is `1`, the top edge of the child view is aligned to the bottom edge of the top target.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for the bottom target. <br /> When the value is `0`, the bottom edge of the child view is aligned to the top edge of the bottom target.<br/>When the value is `1`, the bottom edges of the bottom target and child view are aligned.
+| `HorizontalAlignment`  | RelativeLayout.Alignment| Gets or sets the horizontal alignment of the child view.|
+| `VerticalAlignment`  | RelativeLayout.Alignment| Gets or sets the vertical alignment of the child view.|
+| `FillHorizontal`  | bool | Gets or sets the boolean value indicating whether the child fills its horizontal space or not.|
+| `FillVertical`  | bool | Gets or sets the boolean value indicating whether the child fills its vertical space or not.|
+
+## Target and relativeOffset
+
+ The `Target` and the `RelativeOffset` properties determine the size of layout space. The `Target` indicates the target view in which each edge of the layout space is aligned. The default value is `null`, which refers to the parent's relative layout.
+
+The `RelativeOffset` moves each edge of layout space in proportion to the target view size:
+- If the value is `0.0f`, the edge of the layout space is aligned with the left or top edge of the target view.
+- If the value is `1.0f`, the edge of the layout space is aligned with the right or bottom edge of the target view:
+
+![RelativeLayoutTargetRelativeOffset](./media/relativeLayoutTargetRelativeOffset.png)
+
+## Alignment
+
+ The `HorizontalAlignment` and the `VerticalAlignment` properties describe how to align the child's view with the layout space. The default value is `Start` for both axes:
```

Comment:
Child view or Child's view. I think the noun is incorrect as a Child view.  it can probably be Child's view, Child itself cannot be view. 

If you don't agree, I will go with what you say. 

![image](https://user-images.githubusercontent.com/43839737/107472400-e5394680-6b94-11eb-8f43-f0aec137693b.png)

Reply:
If a layout contains other component in it, we call it child.
So if a view is contained in a layout, we usually call it child view, in developer document.

I think  you'd better refer  https://developer.android.com/s/results?q=child%20view, or google search result of "child view" too.

Reply:
We can use 'child view' here.